contracts/src/bridge/Bridge.sol

```diff
@@ -114,15 +122,15 @@ contract Bridge is OwnableUpgradeable, DelegateCallAware, IBridge {
     ) external override returns (bool success, bytes memory returnData) {
         if (!allowedOutboxesMap[msg.sender].allowed) revert NotOutbox(msg.sender);
         if (data.length > 0 && !to.isContract()) revert NotContract(to);
-        address prevOutbox = activeOutbox;
-        activeOutbox = msg.sender;
+        address prevOutbox = _activeOutbox;
+        _activeOutbox = msg.sender;
```

Comment:
we could add in a comment to explain that activeOutbox will never be `type(uint160).max` or `0` because of this